recipes/veracitools/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "veracitools" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "422764b1e8eec3023212f6640b40c120b5d93dad5f17fa8d2fca34d994536df7"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - veracitools
+  requires:
+    - pytest
```

Comment:
As you are just importing ``veracitools`` it is not necessary to add the ``requires`` ``pytest`` here. Coud you please delete them?

Reply:
I just saw that this project indeed need ``pytest`` to run. So it needs to be in ``run`` then